jetty-http/src/main/java/org/eclipse/jetty/http/HttpParser.java

```diff
@@ -1945,15 +1939,15 @@ public default void badMessage(BadMessageException failure)
          * @param reason the response reason phrase
          * @return true if handling parsing should return
          */
-        public boolean startResponse(HttpVersion version, int status, String reason);
+        boolean startResponse(HttpVersion version, int status, String reason);
     }
 
     /* ------------------------------------------------------------------------------- */
     /* ------------------------------------------------------------------------------- */
     /* ------------------------------------------------------------------------------- */
     public interface ComplianceHandler extends HttpHandler
     {
-        public default void onComplianceViolation(HttpCompliance compliance, HttpComplianceSection violation, String details)
+        default void onComplianceViolation(HttpCompliance compliance, HttpCompliance.Violation violation, String details)
```

Comment:
This is not specific to HttpCompliance, violation reports can come from (currently) HttpParser, CookieCutter, Request.